tekton/ci/shared/eventlistener.yaml

```diff
@@ -27,7 +27,8 @@ spec:
           params:
             - name: "filter"
               value: >-
-                body.repository.full_name.startsWith('tektoncd/') &&
+                (body.repository.full_name.startsWith('tektoncd/') || 
+                (body.repository.full_name.startsWith('tektoncd-catalog/')) &&
```

Comment:
yeah, I think this is the only thing we need to support more than one github org